### DISCUSSION TITLE
Futuro da Impressão 3D

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#493] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/futuro-da-impressao-3d/

**Comment by brunoaugusto** [comment]
![null](<https://camo.githubusercontent.com/274d905eb0746b70b7c81f4a8a1aed8aaed0a248fd81189c8f28a8cf450c66ab/68747470733a2f2f692e696d6775722e636f6d2f743046694a77462e706e67>)

**Comment by maryon-telvanni** [comment]
> [Bruno Augusto](</perfil/?user=brunoaugusto>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/videos/futuro-da-impressao-3d/#comment_1174235444>):

> [![null](<https://camo.githubusercontent.com/274d905eb0746b70b7c81f4a8a1aed8aaed0a248fd81189c8f28a8cf450c66ab/68747470733a2f2f692e696d6775722e636f6d2f743046694a77462e706e67>)](<https://camo.githubusercontent.com/274d905eb0746b70b7c81f4a8a1aed8aaed0a248fd81189c8f28a8cf450c66ab/68747470733a2f2f692e696d6775722e636f6d2f743046694a77462e706e67>)

Nem brinca com uma coisa dessas :scream:
